lib/rubocop/cop/layout/case_indentation.rb

```diff
@@ -3,10 +3,10 @@
 module RuboCop
   module Cop
     module Layout
-      # This cop checks how the *when*s of a *case* expression
-      # are indented in relation to its *case* or *end* keyword.
+      # This cop checks how the __when__s of a _case_ expression
```

Comment:
I think we should always use backticks in such cases. It's more consistent and it works in AsciiDoc.

Reply:
Sure!

Reply:
I updated code representation using backticks.